src/SBAdd.cpp

```diff
@@ -43,6 +43,19 @@ namespace galsim {
         return static_cast<const SBAddImpl&>(*_pimpl).getObjs();
     }
 
+    std::string SBAdd::SBAddImpl::serialize() const
+    {
+        std::ostringstream oss(" ");
+        oss << "galsim._galsim.SBAdd([";
+        ConstIter sptr = _plist.begin();
+        oss << sptr->serialize();
+        for (++sptr; sptr!=_plist.end(); ++sptr) oss << ", " << sptr->serialize();
+        oss << "], galsim.GSParams("<<*gsparams<<"))";
+        return oss.str();
+    }
+
+    // We can't rely on SBProfileImpl::repr for SBAdd b/c it will end up calling the
+    // summands serialize method instead of repr method.  So we override here.
```

Comment:
I don't understand this.  Isn't serialize the same as repr for the SB types?  So why do we need this distinction?